MapboxSearch/sdk/src/main/java/com/mapbox/search/ForwardSearchOptions.kt

```diff
@@ -362,5 +362,8 @@ internal fun ForwardSearchOptions.mapToCore(): CoreSearchOptions = CoreSearchOpt
     route = null,
     sarType = null,
     timeDeviation = null,
-    addonAPI = unsafeParameters?.let { (it as? HashMap) ?: HashMap(it) }
+    addonAPI = unsafeParameters?.let { (it as? HashMap) ?: HashMap(it) },
+    ensureResultsPerCategory = null,
+    // TODO FIXME should we support attribute sets for the forward geocoding
+    attributeSets = null,
```

Comment:
This is coming in an upcoming searchbox release.

Reply:
Let's leave it as is for now. It's not required in this patch